sdk/storage/azfile/internal/testcommon/clients_auth.go

```diff
@@ -39,6 +42,15 @@ const (
 	FakeStorageURL     = "https://fakestorage.file.core.windows.net"
 )
 
+var (
+	SampleSDDL = `O:S-1-5-32-548G:S-1-5-21-397955417-626881126-188441444-512D:(A;;RPWPCCDCLCSWRCWDWOGA;;;S-1-0-0)`
```

Comment:
This is not used anywhere?

Reply:
This is used in `TestSharePutAndGetPermission` in share client.